views/listings/index.erb

```diff
@@ -1,3 +1,14 @@
-<form action="/makersbnb/listings/add" method='get'>
+<h1>Welcome to MakersBnB</h1>
+
+  <% @listings.each do |listing| %>
+    <li>
+        <%= listing.name %>
+        <%= listing.description %>
+        <%= listing.price %>
+    </li>
+  <% end %>
+
```

Comment:
Ever so minor thing. I'd wrap the each loop in unordered-list tags, just so the list tag aren't sitting by themselves. otherwise we're more than likely gonna get some weird formatting going on.

Reply:
Cool. I've just made those changes and pushed again.

Reply:
If it's merged, will it merge with the new changes or do I need to do another pull request?